src/po/sr.po

```diff
@@ -2457,29 +2185,115 @@ msgstr "E324: PostScript излазни фајл не може да се отв
 msgid "E456: Can't open file \"%s\""
 msgstr "E456: Фајл \"%s\" не може да се отвори"
 
-msgid "E456: Can't find PostScript resource file \"prolog.ps\""
-msgstr "E456: PostScript resource фајл \"prolog.ps\" не може да се пронађе"
+msgid "E456: Can't find PostScript resource file \"prolog.ps\""
+msgstr "E456: PostScript resource фајл \"prolog.ps\" не може да се пронађе"
+
+msgid "E456: Can't find PostScript resource file \"cidfont.ps\""
+msgstr "E456: PostScript resource фајл \"cidfont.ps\" не може да се пронађе"
+
+#, c-format
+msgid "E456: Can't find PostScript resource file \"%s.ps\""
+msgstr "E456: PostScript resource фајл \"%s.ps\" не може да се пронађе"
+
+#, c-format
+msgid "E620: Unable to convert to print encoding \"%s\""
+msgstr "E620: Није могућа конверзија у кодирање за штампу \"%s\""
+
+msgid "Sending to printer..."
+msgstr "Слање штампачу..."
+
+msgid "E365: Failed to print PostScript file"
+msgstr "E365: PostScript фајл није успео да се одштампа"
+
+msgid "Print job sent."
+msgstr "Задатак штампе је послат"
+
+msgid "E679: recursive loop loading syncolor.vim"
+msgstr "E679: Рекурзивна петља код учитавања syncolor.vim"
+
+#, c-format
+msgid "E411: highlight group not found: %s"
+msgstr "E411: група истицања није пронађена: %s"
+
+#, c-format
+msgid "E412: Not enough arguments: \":highlight link %s\""
+msgstr "E412: Нема довољно аргумената: \":highlight link %s\""
+
+#, c-format
+msgid "E413: Too many arguments: \":highlight link %s\""
+msgstr "E413: Сувише аргумената: \":highlight link %s\""
+
+msgid "E414: group has settings, highlight link ignored"
+msgstr "E414: група има поставке, highlight link се игнорише"
+
+#, c-format
+msgid "E415: unexpected equal sign: %s"
+msgstr "E415: неочкиван знак једнакости: %s"
+
+#, c-format
+msgid "E416: missing equal sign: %s"
+msgstr "E416: недостаје знак једнакости: %s"
+
+#, c-format
+msgid "E417: missing argument: %s"
+msgstr "E417: недостаје аргумент: %s"
+
+#, c-format
+msgid "E418: Illegal value: %s"
+msgstr "E418: Неважећа вредност: %s"
+
+msgid "E419: FG color unknown"
+msgstr "E419: Непозната FG боја"
```

Comment:
Haha... "E419: Nepoznata PR boja", as in "prednja". :D I'm just sharing a funny thought, not an actual review comment.

Reply:
I forgot to resolve this, it's a leftover. I translated Foreground as prednji plan in other places. I just wonder who would know what does the PP means :-D

Reply:
"Boja prednjeg plana" could be understandable, though "prednji plan" is not a common word either way. I'd say leave it as is.